### DISCUSSION TITLE
Tailor entry as form field for a box

### DISCUSSION THREAD

**Discussion by hmsuchier** [OFFLINE-GmbH/oc-boxes-support#76] [Ideas and feature requests]
Hi,
 Would it be possible to have a form field type to simply pick a tailor entry, the same way as  Tailor's `entries` type, with a dropdown or an autocomplete widget?
Thanks!

**Comment by tobias-kuendig** [comment]
Can you give me some more details about your use-case?

The is the [pagefinder](https://docs.octobercms.com/3.x/element/form/widget-pagefinder.html) widget that can be used to select Tailor models.

**Reply by hmsuchier** [reply]
Thanks for your reply. I'd like users to pick a custom subset of a given Tailor entry type to be displayed in a special way (hence the box and its partial that manage the "special way"). Not within a category, nor something that can be filtered for example: user freely picks this entity, this one and this other one, can reorder them the way they want, and we're done. This can (should) be repeated with several similar boxes.
Thanks!

**Reply by tobias-kuendig** [reply]
Starting from version 3.0.1 you can relate Tailor Models to your Boxes. Since the use-cases here are endless, you'll have to do some manual work. You can find an example in the docs that shows how you would use a recordfinder to pick a related model for a Box: https://docs.boxes.offline.ch/use-cases/including-tailor-content.html#using-a-record-finder

Maybe this helps you to get you use-case solved!

**Reply by hmsuchier** [reply]
Thanks for your answer, I'll look at this!
